### PR TITLE
Jiayou.hello_action version 1.0

### PR DESCRIPTION
**Pull request by epmpub**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109516&drop=dogfoodAlpha

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]
Missing VC Redist:
image

**Comment by epmpub** [comment]
image

the dependency  on  VCRUNTIME140.dll has been fix.

reference:
https://github.com/rust-lang/rust/issues/100874

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> the dependency on VCRUNTIME140.dll has been fix.

This will require dependency support to come out of experimental features and enabled in pipelines

You'll need to include the 2015-2022 if this portable app requires it.

**Comment by epmpub** [comment]
> > the dependency on VCRUNTIME140.dll has been fix.
> 
> This will require dependency support to come out of experimental features and enabled in pipelines
> 
> You'll need to include the 2015-2022 if this portable app requires it.

Yes.  include Dependency package i.e Visual C++ Redistributable is good idea.
i no ideal how to write my GitHub Action .yml file.
so i just use  static link vcruntime.

**Comment by epmpub** [comment]
If you just want to static link vcruntime (and not the whole CRT), [here's a crate](https://crates.io/crates/static_vcruntime) for that. It's used from a build script (build.rs).

Though if you do want to statically link everything then only follow the instructions at the bottom:

In the same directory as your Cargo.toml, create a folder called .cargo. In that folder create the file config.toml and add the following:

[target.'cfg(all(windows, target_env = "msvc"))']
rustflags = ["-C", "target-feature=+crt-static"]

File: manifests/j/Jiayou/hello_action/1.0/Jiayou.hello_action.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Jiayou.hello_action
+PackageVersion: 1.0
+Installers:
+- InstallerUrl: https://github.com/epmpub/hello_action/releases/download/vrefs%2Fheads%2Fmaster/hello_action.exe
+  Architecture: x64
+  InstallerType: portable
+  InstallerSha256: B2E5882D0ED4812C7D97B3A219E4683A9F5D60196BD2D592934F21DD16D5C961
```
Comment (ItzLevvie) [inline manifests/j/Jiayou/hello_action/1.0/Jiayou.hello_action.installer.yaml:10]:
```suggestion
  InstallerSha256: 2831d8059c38aac0b3a6c7908738fe800d3a8fca9cbb6519f83971fc2ed491a2
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Ah I see it works now. Just need to update InstallerSha256 in the manifest

image
image

**Pull request closed without merging**